sstables/sstables.cc

```diff
@@ -3009,6 +3009,19 @@ utils::hashed_key sstable::make_hashed_key(const schema& s, const partition_key&
     return utils::make_hashed_key(static_cast<bytes_view>(key::from_partition_key(s, key)));
 }
 
+future<> sstable::wipe_storage(const sstring& name) noexcept {
```

Comment:
let's s/name/toc_name/ so the interface will make sense to the human reader.

Reply:
It's temporary state. One of next patches makes it `filesystem_storage::wipe(const sstable&)` and `wipe(sstring name)` stops to exist at all

Reply:
ah, I see